### ISSUE TITLE
feat: add badge to search for packages that have no code

### ISSUE THREAD

**Issue by okikio** [jsr-io/jsr#757]
There are a number of devs testing and trying out jsr for the first time, and sometimes it takes a while for projects from npm to eventually be pushed to jsr. It would be pretty convenient if said packages had a badge indicating that they have no-code yet, making it easier to find projects.

**Comment by lucacasonato** [comment]
Yeah, generally they will also have low score. I think search ranking based on score would already help significantly here.

**Comment by KnorpelSenf** [comment]
Perhaps this should be renamed to "feat: include score in search ranking" or perhaps it should just be closed in favour of #312?
